specification/frontdoor/resource-manager/Microsoft.Network/preview/2018-08-01-preview/examples/FrontdoorRedirectRoutingRuleCreate.json

```diff
@@ -0,0 +1,136 @@
+{
+  "parameters": {
+    "api-version": "2018-08-01",
+    "subscriptionId": "subid",
+    "resourceGroupName": "rg1",
+    "frontDoorName": "frontDoor1",
+    "routingRuleName": "redirectRoutingRule1",
+    "routingRuleParameters": {
+      "name": "redirectRoutingRule1",
+      "properties": {
+        "routeType": "Redirect",
+        "frontendEndpoints": [
+          {
+            "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/frontDoors/frontDoor1/frontendEndpoints/frontendEndpoint1"
+          },
+          {
+            "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/frontDoors/frontDoor1/frontendEndpoints/default"
+          }
+        ],
+        "acceptedProtocols": [
+          "Https"
+        ],
+        "patternsToMatch": [
+          "/*"
+        ],
+        "redirectConfiguration": {
+          "redirectType": "Moved",
+          "destinationProtocol": "Https",
+          "destinationHost": "www.bing.com",
+          "destinationPath": "/api",
+          "preservePath": "No",
+          "preserveQueryString": "Yes"
+        },
+        "enabledState": "Enabled"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/frontDoors/frontDoor1/redirectRoutingRule1",
+        "name": "redirectRoutingRule1",
+        "properties": {
+          "routeType": "Redirect",
+          "frontendEndpoints": [
+            {
+              "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/frontDoors/frontDoor1/frontendEndpoints/frontendEndpoint1"
+            },
+            {
+              "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/frontDoors/frontDoor1/frontendEndpoints/default"
+            }
+          ],
+          "acceptedProtocols": [
+            "Https"
+          ],
+          "patternsToMatch": [
+            "/*"
+          ],
+          "redirectConfiguration": {
+            "redirectType": "Moved",
+            "destinationProtocol": "Https",
+            "destinationHost": "www.bing.com",
+            "destinationPath": "/api",
+            "preservePath": "No",
+            "preserveQueryString": "Yes"
```

Comment:
Is it necessary to update the forwarding creation/update examples, to reflect the "routeType" and null "redirectionConfiguration"?
Is "routeType" a required field?

Reply:
I see "routeType" must be optional to support backward compatibility